### ISSUE TITLE
Digitising function

### ISSUE THREAD

**Issue by Andreasr28** [GDI-HE/Mapbender2.8#15]
About the function object digitisation (Geoportal Hessen applikation) to create:
- points
- lines
- polygons

Here it is about creating points and allocate different symbols for visualisation.
- standard symbols
- free symbols

Issue:
Createing points works but change symbols doesn't.
Destination: 
Create points of interest and change symbols between standard and free.

**Comment by Andreasr28** [comment]
To verify i startet digitising points in different browsers:

- i tryed guest user and registered users
- startet the viewer, startet digitisation and creates points 
- in Firefox, Opera, Chrome, Edge, Vivaldi
- the result is:
  - all browsers support the function to create points
  - Edge, Chrome, Vivaldi, Opera -> doesn't show the standard symbol and doesn't show any selected symbol
  - only Firefox shows the standard symbol

**Comment by Andreasr28** [comment]
I have changed the label, this issue is not that important and works in core function.

**Comment by Andreasr28** [comment]
Revalued!
I am currently trying to create a map for a local application on which a certain area is outlined with a track and various points are marked. The whole thing is saved as a .gpx file. 
The information is also basically processed correctly. I can upload my gpx file via <Tools/Digitise Objects> and the outline as well as the points are taken over. If I click on the points, the map is focused correctly. 
Unfortunately, the styling does not work or is not applied when importing. Some of the map applications show manually set styles, others only those from the track, but not the waypoints. 
I have also exported and re-imported working styles to avoid syntax errors. Unfortunately with the same effect.

What else I noticed... If you use the classic version and use GeoJson instead of gpx, the import actually works and is also displayed. So this function already existed. In the modern design, the style of the line is adopted, but not that of the points.
